### PR TITLE
More logs for registry

### PR DESCRIPTION
**Pull request by AleF83**

### CODE REVIEW

File: services/src/modules/registry-schema/resolvers/delete-resources.ts [hunk 1]
```diff
@@ -19,40 +19,53 @@ export default async function (deletions: ResourceGroupMetadataInput, context: R
...
     } finally {
+      logger.trace('Removing policy attachments temporary files...');
       await policyAttachments.cleanup();
     }
 
+    const summary = Object.fromEntries(
+      Object.entries(deletions).map(([k, v]) => [k, v ? (Array.isArray(v) ? v.length : 1) : 0])
+    );
+    logger.info(summary, `Resources were deleted`);
+
```
Comment (yoavkarako) [inline services/src/modules/registry-schema/resolvers/delete-resources.ts:68]: Why remove from the `try` clause?

File: services/src/modules/logger.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ const devLoggerConfig: pino.LoggerOptions = {
   level: 'trace',
   prettyPrint: {
     colorize: true,
-    translateTime: 'HH:MM:ss',
+    translateTime: 'HH:MM:ss.l',
```
Comment (yoavkarako) [inline services/src/modules/logger.ts:14]: Which formatting engine is being used here where `l` correlates to ms?

Comment (AleF83) [reply services/src/modules/logger.ts:14]: `pino` logger uses https://github.com/felixge/node-dateformat

File: services/src/modules/registry-schema/resolvers/delete-resources.ts (hunk 1 above)
Comment (AleF83) [reply services/src/modules/registry-schema/resolvers/delete-resources.ts:68]: This logs end of operation. So for exact time measurement it should be done after `finally` clause.

Comment (yoavkarako) [reply services/src/modules/registry-schema/resolvers/delete-resources.ts:68]: Do you want to log even if there's an exception and the process fails?

Comment (AleF83) [reply services/src/modules/registry-schema/resolvers/delete-resources.ts:68]: If exception will be thrown this code won't run.

**Review by yoavkarako — APPROVED** [review]

**Review by tomeresk — APPROVED** [review]

**Pull request merged**